css/styles.css

```diff
@@ -480,27 +481,32 @@ textarea {
 }
 
 @media (min-width: 780px) {
-  header {
+  .nav-menu {
     position: fixed;
```

Comment:
Your nav menu is currently out of the page. kindly make sure you enclose your items in a flexible container as stated earlier

![image](https://user-images.githubusercontent.com/43564324/165985219-6c4a15f7-414e-4a6a-8d78-1936151d2892.png)